boards/esp32s2-wemos-mini/doc.txt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2022 Benjamin Valentin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_esp32s2-wemos-mini Wemos S2 mini Board
+ * @ingroup     boards_esp32s2
+ * @brief       Support for Wemos S2 mini Board
+ * @author      Benjamin Valentin <benjamin.valentin@ml-pa.com>
+
+\section esp32s2-wemos-mini Wemos S2 mini
+
+## Overview {#esp32s2-wemos-mini_overview}
+
+\image html "https://www.wemos.cc/en/latest/_images/s2_mini_v1.0.0_4_16x9.jpg" "Wemos S2 mini" width=600px
+
+Documentation for the board can be found on the vendor page:
+
+- [Wemos S2 mini](https://www.wemos.cc/en/latest/s2/s2_mini.html)
+- [Schematic](https://www.wemos.cc/en/latest/_static/files/sch_s2_mini_v1.0.0.pdf)
+
+## Flashing the Device {#esp32s2-wemos-mini_flashing}
+
+Flashing RIOT is quite easy. The board has a Micro-USB connector with
+reset/boot/flash logic. Just connect the board to your host computer
```

Comment:
According to the schematic, the D- and D+ signals are directly connected to the USB interface of the ESP32-C3 GPIO19, GPIO20. There is no reset logic connected to the EN pin that triggers a short LOW impuls by toggling DTR and RTS. So I'm wondering how the flashing works at all?